apel/db/loader/loader.py

```diff
@@ -146,6 +146,17 @@ def load_all_msgs(self):
 
             try:
                 log.info("Loading message %s. ID = %s", self.current_msg, msg_id)
+
+                if (
+                    not data or
```

Comment:
Similar issue with indentation to this: https://github.com/apel/ssm/pull/228#discussion_r1231209359

Could we go with something like this (from [PEP 8](https://peps.python.org/pep-0008/)) with the line breaks before the logical operator:

```python
# Add some extra indentation on the conditional continuation line.
if (this_is_one_thing
        and that_is_another_thing):
    do_something()
```

Reply:
Done. Updated the codebase